index.html

```diff
@@ -10,951 +12,1067 @@
+              <label for="emailInput" class="control is-expanded visuallyhidden">Enter your email</label>
+              <input aria-labelledby="subscribtion" name="emailInput" id="emailInput">
+              <button aria-labelledby="subscribtion" id="subscribeButton" name="subscribeButton" type="submit" class="button is-info">Notify Me</button>
+            </fieldset>
+          </form>
         </div>
       </div>
-    </div>
-  </div>
+    </section>
+  </header>
+	<main id="maincontent" role="main">
+		<div id="crumbs_frame" class="container" aria-label="breadcrumbs">
+			<ul class="bct">
+				<!-- .bct / Breadcrumbs -->
+				<li><a href="./" aria-label="breadcrumbs to main page">MainPage</a><span class="cr">»</span></li>
+				<li><a href="index.html#nav" aria-label="breadcrumbs to pictures">Pictures</a><span class="cr">»</span></li>
+				<li><a href="index.html#Services" class="current" aria-current="page" aria-label="breadcrumbs to services">Services</a></li>
+			</ul>
+		</div>
+		<article class="section" id="about">
+			<section class="section-heading has-text-centered" aria-label="About us">
+				<h1 class="title is-2">About Us</h1>
+				<h2 class="subtitle is-5">We of all trades, master of "some"</h2>
+				<div class="container">
+					<p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
+						field of
+						<b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
+						Looking for an opportunity to work and upgrade, as well as being involved in an organization that
+						believes
+						in gaining a competitive edge and giving back to the community.</p>
+				</div>
+			</section>
 
-  <div class="section" id="about">
-    <div class="section-heading has-text-centered">
-      <div class="title is-2">About Us</div>
-      <div class="subtitle is-5">We of all trades, master of "some"</div>
-      <div class="container">
-        <p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
-          field of
-          <b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
-          Looking for an opportunity to work and upgrade, as well as being involved in an organization that
-          believes
-          in gaining a competitive edge and giving back to the community.</p>
-      </div>
-    </div>
+			<div id="mainsection" class="content container">
+				<section aria-label="Hello World">
+					<h2 class="is-1 title">Hello World</h2>
+					<p>Lorem ipsum<sup><a tabindex="0" role="link">[1]<span class="visuallyhidden">&nbsp;footnote in dictionary</span></a></sup> dolor sit amet, consectetur adipiscing elit. Nulla accumsan, metus ultrices eleifend gravida, nulla nunc varius lectus, nec rutrum justo nibh eu lectus. Ut vulputate semper dui. Fusce erat odio, sollicitudin vel erat vel, interdum mattis neque. Sub<sub>script</sub> works as well!</p>
```

Comment:
If we use the **<a** tag without the **href** attribute, the tag still has the `role="link"` by default

<img width="895" alt="image" src="https://user-images.githubusercontent.com/16785774/172355609-d3a49c23-f185-47ee-8d3f-064ed5314813.png">

Reply:
Yea, but NVDA will still not see it in "insert+F7" mode. In the list of all links will be only links with HREF attribute.